xbmc/media/decoderfilter/DecoderFilterManager.cpp

```diff
@@ -122,30 +123,30 @@ bool CDecoderFilterManager::Load()
   if (!XFILE::CFile::Exists(fileName))
     return true;
 
-  CLog::Log(LOGINFO, "{}: loading filters from {}", CLASSNAME, fileName);
+  CLog::LogF(LOGINFO, "loading filters from {}", fileName);
```

Comment:
Just a heads up that `static const char* CLASSNAME = "CDecoderFilter";` (line 35) gets unused after this PR:
https://jenkins.kodi.tv/job/android-x86-docker/2350/clang/

There are also a couple of other issues (LDFLAGS and INCLUDES) that I guess have been introduced with some recent cmake change (tinyxml2?)